src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
```

Comment:
Prefer `const` over `let` for the `result` variable if you are not reassigning it. In this case, you can calculate the result in a single expression without reassigning the variable.